pkg/storaged/storage.scss

```diff
@@ -47,8 +47,8 @@
     padding-bottom: var(--pf-global--spacer--md);
 }
 
-#detail-content, #detail-content .pf-c-card {
```

Comment:
Maybe you want to remove the rule here as well? And then it won't be necessary to remove the table padding https://github.com/cockpit-project/cockpit/blob/master/pkg/lib/ct-card.scss#L9 ihis rule sets a xs bottom padding for the cards that contain tables. I feel it's more straightforward to align the tbody with the bottom of the card and not care about it's border any more.

Reply:
I agree.

(BTW, in case anyone's wondering about the `--pf-global--spacer--xs`: I added the tiny bit of padding at the bottom and dropped the bottom line as the cards with tables felt oddly short without it. Adding the tiny padding makes them feel balanced. It has to do with visual weight; when designing and drawing stuff, sometimes you need to fudge things a little to have them feel correct. That's why it's xs instead of 0.)

Reply:
Oh I removed this, I felt it's not right. I can get it back then.

Reply:
OK. We can keep it 0.